TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/WorkforceData/TpsCsvExtractFileImporter.cs

```diff
@@ -141,12 +141,16 @@ INSERT INTO tps_csv_extracts (
                 loadErrors = loadErrors | TpsCsvExtractItemLoadErrors.GenderIncorrectFormat;
             }
 
-            if (row.MemberEmailAddress is not null && !EmailAddress.TryParse(row.MemberEmailAddress, out _))
+            var memberEmailAddress = row.MemberEmailAddress?.Trim();
+            memberEmailAddress = memberEmailAddress?.Length > 0 ? memberEmailAddress : null;
+            if (memberEmailAddress is not null && !EmailAddress.TryParse(memberEmailAddress, out _))
```

Comment:
Would the following be more succinct?
```
if (!memberEmailAddress.IsNullOrWhiteSpace() && !EmailAddress.TryParse(memberEmailAddress, out _))
```

Reply:
I hear you but I still need the trimmed value to use later though hence trimming it first and using a variable